R/plumber-step.R

```diff
@@ -20,12 +20,23 @@ resetForward <- function() {
   exec$forward <- FALSE
 }
 
+#' plumber step R6 class
+#' @description an object representing a step in the lifecycle of the treatment
+#' of a request by a plumber router.
 PlumberStep <- R6Class(
   "PlumberStep",
   inherit=hookable,
   public = list(
+    #' @field lines lines from step block
     lines = NA,
+    #' @field serializer step serializer function
     serializer = NULL,
+    #' @description Create a new `PlumberStep` object
```

Comment:
Ah, just remembered.  Can you change all  `` `func` `` to `[func()]`.  This will allow for pkgdown to link to the proper help pages for free.

```suggestion
    #' @description Create a new [PlumberStep()] object
```

(I forgot in my earlier changes above)